config/config.js

```diff
@@ -293,6 +296,24 @@ c.substitution.docker.volume.mode = ":ro";
 c.substitution.docker.cmd = 'docker run -it --rm';
 c.substitution.docker.imageNamePrefix = 'erc:';
 
+c.dns = {};
+c.dns.dnsmasq = {};
+c.dns.priority = {};
+c.dns.dockerfile = "" +
+    "FROM alpine:edge\n" +
+    "RUN apk --no-cache add dnsmasq\n" +
+    "EXPOSE 53/tcp 53/udp\n" +
+    "COPY dnsmasq.conf /etc/dnsmasq.conf\n" +
+    "CMD [\"dnsmasq\", \"--no-daemon\"]";
+c.dns.dnsmasq.default = "" +
+    "log-queries\n" +
+    "no-hosts\n" +
+    "no-resolv\n" +
+    "cache-size=100000\n";
+c.dns.dnsmasq.filterDummy = "server=/";
+c.dns.priority.publisher = 100;
+c.dns.priority.journal = 50;
```

Comment:
Gibt es hier irgendwas das ein Plattformbetreiber Konfigurieren wollen würde, oder ist das ziemlich stabil? Wenn Konfiguration, dann gerne das Muster mit den Umgebungsvariablen nutzen wie an anderen Stellen in dieser Datei.